src/schema/nmdc.yaml

```diff
@@ -324,13 +335,30 @@ classes:
       - websites
       - publications
       - type  # custom slot that specifies object type
+      # external IDs
+      - INSDC bioproject identifiers
+      - INSDC SRA ENA study identifiers
+      - GOLD study identifiers
+      - MGnify project identifiers
     slot_usage:
       doi:
         description: >-
           The dataset citation for this study
+    exact_mappings:
+      - SIO:001066  ## study
+      - NCIT:C63536 ## study
+      - OBI:0000066 ## investigation
+      - ISA:Study
+    close_mappings:
+      - NCIT:C41198 ## investigation
```

Comment:
Why is `NCIT:C41198 ## investigation` a `close_mapping` but `OBI:0000066 ## investigation` and `exact_mapping`?

Definition for `NCIT:C41198`:
```
 The act or process of a systematic and thorough examination; research, study; the process of inquiring into or following up, intended to develop facts.
```
Definition for `OBI:0000066`:
```
A planned process that consists of parts: planning, study design execution, documentation and which produce conclusion(s).
```

Reply:
NCIT has both investigation and study, exact mappings should be one-to-one

Reply:
Perhaps this just points to a weakness in the NCIT? It might be better to not map to NCIT:Investigation.